ddtrace/llmobs/_llmobs.py

```diff
@@ -105,16 +106,20 @@ def __init__(self, tracer=None):
         self.tracer = tracer or ddtrace.tracer
         self._llmobs_context_provider = LLMObsContextProvider()
         self._llmobs_span_writer = LLMObsSpanWriter(
-            is_agentless=config._llmobs_agentless_enabled,
-            agentless_url="%s.%s" % (AGENTLESS_BASE_URL, config._dd_site),
+            site=config._dd_site,
```

Comment:
One thing I am unhappy with is that `site` is an arg, but for agentless writers we depend solely on the `_agentless_url`. Need to consolidate this to use `site`.